From a8d167a987b6063f1d8ad0a1dbae018640103243 Mon Sep 17 00:00:00 2001
From: ahs <amrita.h.s@intel.com>
Date: Thu, 1 Sep 2022 13:34:35 +0530
Subject: [PATCH] Blanket enable avx for all math APIs

Build with -avx, -avx2 and -fma flags

Tracked-On: OAM-103681
Signed-off-by: ahs <amrita.h.s@intel.com>
---
 Android.bp | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/Android.bp b/Android.bp
index ba814eb..cd6e674 100755
--- a/Android.bp
+++ b/Android.bp
@@ -85,6 +85,11 @@ cc_library_static {
                 "-DHAVE_FAST_FMA=1",
             ],
         },
+        x86_64: {
+            cflags: [
+                "-mavx", "-mavx2", "-mfma",
+            ],
+        },
     },
 
     target: {
-- 
2.17.1

